google/cloud/internal/async_retry_unary_rpc_test.cc

```diff
@@ -38,6 +38,7 @@ using ::testing::HasSubstr;
 using ::testing::Return;
 using Request = google::protobuf::Timestamp;
 using Response = google::protobuf::Duration;
+using namespace std::chrono_literals;
```

Comment:
I'm not sure this is right, because the `std` chrono literals are like `10us`, not the `10_us` we get from google/cloud/testing_util/chrono_literals.h.  Did this actually fix things in the import?

Reply:
It's still building... 

I'll update this if (or when) it fails :(

Reply:
Didn't like that even when I changed then to 10us

Trying the 10_us -> us(10)

Reply:
Built in google3!